app/components/common/committeeListing/index.js

```diff
@@ -81,6 +83,29 @@
         rowHeight: 40,
         suppressMovableColumns: true
       };
+
+      // Combine first and last name, capitalize
+      function getFriendlyName(resource) {
```

Comment:
Yeesh. . . tough crowd. Alright, I'll take it out. I agree with the principle here, but it makes the tables look messy.